tools/istio-iptables/pkg/dependencies/implementation.go

```diff
@@ -90,20 +93,42 @@ func (r *RealDependencies) execute(cmd string, ignoreErrors bool, args ...string
 	return externalCommand.Run()
 }
 
-func (r *RealDependencies) executeXTables(cmd string, ignoreErrors bool, args ...string) error {
+func (r *RealDependencies) executeXTables(cmd string, ignoreErrors bool, args ...string) (err error) {
 	if r.CNIMode {
 		originalCmd := cmd
 		cmd = constants.NSENTER
 		args = append([]string{fmt.Sprintf("--net=%v", r.NetworkNamespace), "--", originalCmd}, args...)
 	}
 	log.Infof("Running command: %s %s", cmd, strings.Join(args, " "))
-	externalCommand := exec.Command(cmd, args...)
-	stdout := &bytes.Buffer{}
-	stderr := &bytes.Buffer{}
-	externalCommand.Stdout = stdout
-	externalCommand.Stderr = stderr
 
-	err := externalCommand.Run()
+	var stdout, stderr *bytes.Buffer
+	b := backoff.NewExponentialBackOff()
```

Comment:
What is the max interval ?
If you are using backoff lib, you can just use the retry function in it which is imo cleaner.